.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Ansible Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
**Nitpick:** do we really want *-latest?

Sure, I can see pros and cons...
But on the other side the ansible version is fixed (in override-deps)
So do we want to be on the latest and greatest everywhere, or nowhere?
I would make everything latest, or everything stable, but not a mixture.

**Note:** in my comments I tag with "Nitpick" stuff that I don't feel strongly about.
Nothing is wrong, it is just a personal opinion.
Implement if you like it, ignore if you don't (there is no need to explain why not :-)

**Disclaimer:** I know nothing about Ansible

Reply:
One more thing: if all my comments are nitpicks, then I usually approve. You can implement or not, your call :-)

Reply:
I think latest is probably fine, but i'm not sure.  The actual ansible version is fixed.

Reply:
Yeah, I think latest is fine, given it's GH Actions.  For applications, there's the balance between updating versions too frequently (instability and deps hell) and too infrequently (security holes, new features inaccessible).  I think the reason why the model GH Actions is quite different is because each job ("task") is run on a fresh VM, and only for a short amount of time.  The actions are written independent of the OS's bundled libs/deps, and thus actions are independent of each other.  So I think that's why the [selection of OS versions](https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) are so few.  Slightly unrelated, but interestingly, for paid situations, there's a [1:2:10 ratio](https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-billing-for-github-actions) of cost among the different OSes.